lib/peer_manager/modules/nrf_ble_lesc.c

```diff
@@ -56,59 +53,68 @@ static bool m_lesc_oobd_own_generated;
 static ble_gap_lesc_oob_data_t m_ble_lesc_oobd_own;
 static nrf_ble_lesc_peer_oob_data_handler m_lesc_oobd_peer_handler;
 
+#define ECC_PUB_KEY_UNCOMPRESSED_FORMAT_MARKER 0x04
+#define COORDINATE_SIZE (BLE_GAP_LESC_P256_PK_LEN / 2)
+
+/* Convert an ECC (secp256r1) public key from between big-endian and little-endian.
+ * The two coordinates are converted individually.
+ */
+static void ecc_public_key_byte_order_invert(const uint8_t *raw_key_in, uint8_t *raw_key_out)
```

Comment:
and here you could perhaps use

```
sys_memcpy_swap(x_out, x_in, COORDINATE_SIZE);
sys_memcpy_swap(y_out, y_in, COORDINATE_SIZE);
```

from the `byteorder.h` header too

Reply:
Thanks! Seems to work well. Updated.